doc/iterations/2024.11/add_support_for_form_property_section_layout.adoc

```diff
@@ -0,0 +1,35 @@
+= Add support for form property section layout
+
+== Problem
+
+A property section in a form is made of a label, a help icon and a edition area.
+Currently, the layout is always vertical: label and help icon are in div and the edition area is below.
+
+== Key Result
+
+It will be possible as a specifier to choose if the layout is horizontal or vertical.
+
+== Solution
+
+A new attribute `flexDirection` is added on form widget styles. The value is among `[row, rowReverse, column, columnReverse]`.
+
+The label and the help icon are in the same "label" div. The `flexDirection` attribute aims to layout the "label" div and the edition div.
+If `row` and `row-reverse` the "label" div will take `1fr` and the edition div `2fr`.
+That will allow to align the "label" div of all the property sections.
```

Comment:
If we uses flexbox, we won't use the `fr` unit which comes from `grid`. It's either flexbox or grid.